massa-ledger-worker/src/ledger.rs

```diff
@@ -166,10 +175,17 @@ impl LedgerController for FinalLedger {
     ///
     /// # Returns
     /// The last key inserted
-    fn set_ledger_part(&self, data: Vec<u8>) -> Result<StreamingStep<Vec<u8>>, ModelsError> {
+    fn set_ledger_part(&self, data: Vec<u8>) -> Result<StreamingStep<Key>, ModelsError> {
         self.sorted_ledger.set_ledger_part(data.as_bytes())
     }
 
+    /// Reset the disk ledger.
+    ///
+    /// USED FOR BOOTSTRAP ONLY
```

Comment:
same remark as previously

Reply:
@bilboquet is this treated in https://github.com/massalabs/massa/pull/3698 ?

Reply:
also related to the point above

Reply:
ok I'll close this discussion in favor of #3698